Makefile

```diff
@@ -166,6 +166,7 @@ check-node:
 PHONY+= smoke-test
 smoke-test: lyra identity puppet-dsl yaml-dsl lyra-plugins generate
 	@echo "🔘 Running a smoke test with 'foobernetes' workflow"
+	@rm deployment.json*
```

Comment:
This fails when the file doesn't exist (which is what's causing the travis build failure).

Reply:
i reverted this change, it was just weirdness in my environment and not something CI will ever see.